sdk/python/kfp/_client.py

```diff
@@ -193,18 +199,39 @@ def _get_url_prefix(self):
     # In-cluster pod. We could use relative URL.
     return '/pipeline'
 
-  def create_experiment(self, name, description=None, namespace=''):
+  def set_user_namespace(self, namespace):
+    """Set user namespace into local context setting file.
```

Comment:
Updated the docstring with callout: "This function should only be used when Kubeflow Pipelines is in the multi-user mode.".
Looks like per current backend implementation, if a namespace value if passed to the backend, it will not be ignored. The namespace value will end up being stored in the experiment table. Maybe we should error out in the backend?

Reply:
yeah sounds good. let's error it out